### PR TITLE
Move code from Travis configuration to scripts

### PR DESCRIPTION
**Pull request by mnonnenmacher**
This makes it easier to run the same checks locally.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/934)

### CODE REVIEW

File: ktlint.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Run ktlint but exclude rules that are currently not applied to this project.
+
+if ./ktlint | grep -Ev "(Wildcard import|Unexpected spacing before)"; then
+  echo "Please correct above lint issues."
+  exit 1
+else
```
Comment (sschuberth) [inline ktlint.sh]: Nit: Could drop the `else` branch here as you're already `exit`ing above.

File: ktlint.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  echo "No relevant lint issues found."
```
Comment (sschuberth) [inline ktlint.sh]: While we're at it, should we say "ktlint issues" here (and also above) to distinguish from lint issues found by detekt?

File: check-copyright.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Check if all Kotlin files contain a copyright statement.
+
+if git grep -L "Copyright" -- "*.kt"; then
+  echo "Please add copyright statements to the above Kotlin files."
+  exit 1
+else
```
Comment (sschuberth) [inline check-copyright.sh]: Nit: Could drop the `else` branch here as you're already `exit`ing above.

File: ktlint.sh (hunk 1 above)
Comment (mnonnenmacher) [reply ktlint.sh]: Technically true, but I usually find it easier to understand the flow if `if` and `else` are clearly marked.

File: ktlint.sh (hunk 2 above)
Comment (mnonnenmacher) [reply ktlint.sh]: Done.

**Comment by mnonnenmacher** [comment]
@sschuberth I think I wouldn't like to call it `ci`, as the intention was to use it locally. What about `scripts` or `tools`? Should we move the `ktlint` binary as well?

**Comment by sschuberth** [comment]
> Should we move the `ktlint` binary as well?

Makes sense to me, and as a result I'd call it `tools`, not `scripts`, as `ktlint` is not a script.

**Comment by mnonnenmacher** [comment]
@sschuberth Done.

File: tools/ktlint.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Run ktlint but exclude rules that are currently not applied to this project.
+
+BASEDIR=$(dirname "$0")
+
+if $BASEDIR/ktlint | grep -Ev "(Wildcard import|Unexpected spacing before)"; then
```
Comment (sschuberth) [inline tools/ktlint.sh]: Nit: Should we now quote the absolute path to `ktlint` here?

**Pull request merged**